src/app/chat/event_handling.nim

```diff
@@ -10,6 +10,8 @@ proc handleChatEvents(self: ChatController) =
   # Display already saved messages
   self.status.events.on("messagesLoaded") do(e:Args):
     let evArgs = MsgsLoadedArgs(e)
+    if evArgs.messages.len > 0:
+      self.status.chat.statusUpdates()
```

Comment:
I'm thinking if it wouldnt be better if instead of having this code here inside on `messagesLoaded`, instead call that function inside `proc init*(self: ChatController)` in src/app/chat/core.nim, so it's only once when you login.